client/internal/routemanager/dynamic/route_ios.go

```diff
@@ -0,0 +1,51 @@
+//go:build ios
+
+package dynamic
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/miekg/dns"
+	log "github.com/sirupsen/logrus"
+
+	nbdns "github.com/netbirdio/netbird/client/internal/dns"
+
+	"github.com/netbirdio/netbird/management/domain"
+)
+
+const dialTimeout = 1 * time.Second
```

Comment:
Maybe the timeout is a bit low for servers via VPN? Speaking of, we don't use any timeout in the regular LookupIP, so we should probably streamline these two